tests/test_ideogram.py

```diff
@@ -214,6 +214,7 @@ def assert_callback(prop_value, nclicks, input_value):
 
     driver = dash_threaded.driver
 
+    driver.implicitly_wait(1)
```

Comment:
Don't do that, it waits for every selenium operation after, it's not a sleep.

Reply:
Thanks for the tip. Now I use the `wait_for_elements_by_css_selector` funciton from pytest_dash instead :)